### ISSUE TITLE
npm audit failure

### ISSUE THREAD

**Issue by umair00** [telus/semantic-release-config#16]
There are some security vulnerabilities in dependencies of this package with `npm audit`:
![image](https://user-images.githubusercontent.com/40245443/67770549-fb1d0980-fa2c-11e9-828b-7ab9b14c650a.png)

**Comment by maumercado** [comment]
We are aware of it since last week, for more info check here https://github.com/semantic-release/semantic-release/issues/1327 

we are waiting for @semantic-release/npm to be updated.

**Comment by pkandathil** [comment]
Is there a workaround because right now we can't deploy code?

**Comment by maumercado** [comment]
\#\# Temporary fix

In case anyone else's deploys are stuck on this - https://www.npmjs.com/package/npm-force-resolutions may help you out while this is addressed, or while your dependencies update their dependencies (which may need to update their dependencies...)

1\. Add this to your package.json

```
"resolutions": {
  "https-proxy-agent": "^2.2.4"
}
```

2\. Add this to your scripts inside package.json

```
"preinstall": "npx npm-force-resolutions"```
```

3\. Let npm-force-resolutions do it's thing
```
rm -r node\_modules
npx npm-force-resolutions
npm install
```

4\. re-run your audit `npm audit`.

This will force your dependencies to update to 2.2.4 as of right now.

Remember to get rid of this workaround as soon as either

 1. The security fix is properly applied to 2.2.x

 2. All of your various dependencies update their versions, which is what we are waiting for on our side.


### However if this does not fix it.

You can rely on audit-ci whitelisting until a proper fix has being created by the maintainers the proper dependencies.

https://github.com/telus/technology-forum/issues/307#issuecomment-511931654
